Source/WebCore/dom/Document.cpp

```diff
@@ -7268,7 +7268,8 @@ bool Document::hasRecentUserInteractionForNavigationFromJS() const
     if (UserGestureIndicator::processingUserGesture(this))
         return true;
 
-    return (MonotonicTime::now() - lastHandledUserGestureTimestamp()) <= UserGestureToken::maximumIntervalForUserGestureForwarding;
+    static constexpr Seconds maximumItervalForUserGestureForwarding { 10_s };
```

Comment:
10s seems like a really long time. How about 4s or 5s?

Reply:
I am trying to be conservative since this is new behavior and we are still way stricter than Chrome on this (as mentioned in my ChangeLog). It seems GitHub is waiting on some kind of networking before calling pushState() so I was particularly worried about slow networking. My proposal would be to use 10s for the branch and we can try to lower it on trunk.